src/ResourceManager/EventHub/Commands.EventHub/Cmdlets/EventHub/RemoveAzureEventHubAuthorizationRules.cs

```diff
@@ -0,0 +1,58 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using System.Management.Automation;
+
+namespace Microsoft.Azure.Commands.EventHub.Commands.EventHub
+{
+
+    [Cmdlet(VerbsCommon.Remove, EventHubAuthorizationRulesVerb)]
+    public class RemoveAzureEventHubAuthorizationRules : AzureEventHubsCmdletBase
+    {
+        [Parameter(Mandatory = true,
+            ValueFromPipelineByPropertyName = true,
+            Position = 0,
+            HelpMessage = "Resource Group Name.")]
+        [ValidateNotNullOrEmpty]
+         public string ResourceGroupName { get; set; }
+
+        [Parameter(Mandatory = true,
+            ValueFromPipelineByPropertyName = true,
+            Position = 1,
+            HelpMessage = "Namespace Name.")]
+        [ValidateNotNullOrEmpty]
+        public string NamespaceName { get; set; }
+
+        [Parameter(Mandatory = true,
+            ValueFromPipelineByPropertyName = true,
+            Position = 2,
+            HelpMessage = "EventHub Name.")]
+        [ValidateNotNullOrEmpty]
+        public string EventHubName { get; set; }
+
+        [Parameter(Mandatory = true,
+            ValueFromPipelineByPropertyName = true,
+            Position = 3,
+            HelpMessage = "EventHub AuthorizationRule Name.")]
+        [ValidateNotNullOrEmpty]
+        public string AuthorizationRule { get; set; }
+
+        public override void ExecuteCmdlet()
+        {
+            // Delete EventHub authorizationRule
+            var deleteAuthRule = Client.DeleteEventHubAuthorizationRules(ResourceGroupName, NamespaceName, EventHubName, AuthorizationRule);
+            WriteObject(deleteAuthRule);
```

Comment:
You should have presentation types that you use as Output types for your cmdlets. This prevents breaking changes in the underlying management library from being breaking changes in your powershell cmdlets.